### ISSUE TITLE
Triple hyphen ("---") disappears from the output

### ISSUE THREAD

**Issue by gennaroprota** [erez-o/docsforge#30]
Hi,

it seems that a sentence like this:

    At the time of this writing---October 30, 2020---I'm reporting an issue here.

is rendered as:

    At the time of this writingOctober 30, 2020I'm reporting an issue here.

by DocsForge, eating up each instance of "---". The issue doesn't appear to be with Doxygen (I tried with both settings for the `MARKDOWN_SUPPORT` option, just in case). An example in my documentation is [here](https://breath.docsforge.com/master/api/breath/crc/) (you can search for "at the time of this writing").

P.S.: perhaps it would be better to link to a specific revision on GitHub in our issue reports? Because, considering e.g. the link to the DocsForge documentation above, nobody could see the problem if they followed the link two years from now, when the issue will have been fixed.

**Comment by erez-o** [comment]
Thanks, I'll check it.

Regarding revision -  yes, if you can write the docsforge version it would be great, you can see it in your [build log](https://breath.docsforge.com/admin/view-build-log/master/).

> task_id: 123f5971-0226-48d6-bddc-bfca15e16cb4, added to queue on 2020-10-30 09:44:06, waited in queue for 0.061165 seconds. Built with docsforge version 3.4.15

I can in the future add a "report bug in build" button on docsforge and the whole setup will be stored for future reference, but for now it's an overkill and there are other pressing issues.

Links always get broken, as long as the bug is fixed, I think there's no reason to follow the bug link two years from now.

**Comment by erez-o** [comment]
Fixed and released on version 3.4.16.

The resulting link you provided is now rendered correctly as:

```
at the time of this writing—October 7, 2020—is in the same...
```

Note that doxygen transforms `---` to `em dash`, at least when `MARKDOWN_SUPPORT: YES`

**Comment by gennaroprota** [comment]
Thanks! Yes, indeed transforming "---" into an em dash is nice: the source code looks cleaner, IMHO, than if you used `&mdash;` (which would make the documentation harder to read directly from the source file), but the HTML output gets the nicer character; so, the best of both worlds.
